config/webpack/production.js

```diff
@@ -0,0 +1,46 @@
+// Note: You must restart bin/webpack-dev-server for changes to take effect
+
+/* eslint global-require: 0 */
+
+const webpack = require('webpack')
+const merge = require('webpack-merge')
+const CompressionPlugin = require('compression-webpack-plugin')
+const BabiliPlugin = require('babili-webpack-plugin')
```

Comment:
Yeah so that doesn't really apply to tree-shaking; tree-shaking is about bundling ES modules, which is handled by Webpack 2 before the rest of the ES6-to-ES5 transpilation. You can test it, but I'm fairly certain Uglify alone (with mangle+compress) will still beat Babili, assuming you're outputting ES5 code. (In fact it looks like [Babili's own benchmarks](https://github.com/babel/babili#benchmarks) still show Uglify winning, although I know they're making progress.)